pytmc/xml_collector.py

```diff
@@ -874,72 +736,122 @@ def valid(self):
                                   for key in self._required_fields)
         return has_required_keys and has_required_fields
 
-    def render_record(self):
+    @property
+    def records(self):
+        """Generated :class:`.EPICSRecord` objects"""
+        return NotImplemented
```

Comment:
```suggestion
        raise NotImplementedError()
```

Returning `NotImplemented` should be just for `__lt__` et al (See https://stackoverflow.com/questions/878943/why-return-notimplemented-instead-of-raising-notimplementederror)